src/wagtail_live/models.py

```diff
@@ -116,7 +123,7 @@ def add_live_post(self, live_post):
 
         # Insert to keep posts sorted by time
         self.live_posts.insert(lp_index, ("live_post", live_post))
-
+        self.last_updated_at = post_created_at
```

Comment:
I think it is odd that for changed and deleted messages, last_updated_at is set to "now" on the model but for new messages, it is set to "post_created_at". I think any value other than now() would be weird here, because it could cause older messages to be shown again later (if an post with created time in the past is added). So essentially you are giving the live post creator the tools to accidentally break it by supplying a different creation time. Currently this is not a problem because the creator also uses now(), but I think it might be more consistent (with "modified" and "last_updated_at") if the "created" value is also set here (to "now") and not received from the live post creator.

Reply:
For changed messages, last_update_ts is also set to post_modified which is set to now.

I plan to submit another PR after this, to add a received_at field to LivePostBlock which will equals to now and use the messaging app value for the created field to keep posts ordered.

Reply:
Yes then we end up with that feature after all. Timestamps are hard :)